README.adoc

```diff
@@ -219,23 +224,23 @@ acceptance tests. They can be run via:
 $ bundle exec rake features
 ---------------------------
 
-_NOTE_: Only Linux OS is supported at present.
+*NOTE:* These tests can only be executed in Linux, at present.
```

Comment:
@hferentschik 
Until this issue https://github.com/projectatomic/vagrant-service-manager/issues/213 is fixed we should keep it.
I don't think current tests will run in Windows except few.